crawl-ref/source/main.cc

```diff
@@ -2331,6 +2331,12 @@ void world_reacts()
     // the loudest noise tracking for the next world_reacts cycle.
     you.los_noise_last_turn = you.los_noise_level;
     you.los_noise_level = 0;
+    // palentongas uncurl at the end of the turn
+    if (you.props[PALENTONGA_CURL_KEY].get_int() > 0)
+    {
+        you.props[PALENTONGA_CURL_KEY] = 0;
```

Comment:
What about making it boolean? Seems this code only ever uses 0/1.

Reply:
For some reason I thought we didn't support bools. Good call.